addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -50,7 +50,10 @@ export class MassMailingHtmlField extends HtmlField {
             resizable: false,
             defaultDataForLinkTools: { isNewWindow: true },
             toolbarTemplate: 'mass_mailing.web_editor_toolbar',
-            onWysiwygBlur: () => this.wysiwyg.odooEditor.toolbarHide(),
+            onWysiwygBlur: () => {
+                this.commitChanges();
```

Comment:
If I understand the commit message correctly, that is basically duplicating the `onWysiwygBlur` from html field then ? If yes it means that if we ever add something else in the `onWysiwygBlur` of the html field then it's not going to be reflected in mass mailing. I think what the code here should do is to call the original `onWysiwygBlur` before calling its own. Or override the `_onWysiwygBlur` method of html field, basically.

Reply:
Well, I get your point and I think that is in fact a good point. However, I do not have the same feeling that changes in the save mechanism of the html field or wysiwyg have often broken the save of `mass_mailing`. Apart from the owl conversion, which was a complete refactor so I don't think it should really count, nothing comes to my mind. Do you have specific examples in mind ?

Reply:
Not for saving specifically but we've had bugs in `mass_mailing` coming from changes made to the field or wysiwyg without considering the weird iframe context of the module, or the limitations of emails. Regarding saving, the fact that `mass_mailing` requires a conversion process on save, which can take a while when there are background images, made the concept of autosaving a little bit inappropriate as is, leading me to think we might want to think twice before coupling it to whatever is done in `wysiwyg` in general.